wca-regulations.md

```diff
@@ -358,13 +358,13 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
-        - A1a2) Cumulative time limits may be enforced (e.g. 3 solves with a cumulative time limit of 20 minutes - see [Regulation A1a5](regulations:regulation:A1a5)). The time elapsed in a DNF result counts towards the cumulative time limit.
-        - A1a3) For each round, any time limits must be announced before the round starts, and should not be changed after it has begun. Changes must be made at the discretion of the WCA Delegate, who must carefully consider the fairness of the change.
-        - A1a4) The competitor must end each solve within the time limit. If a competitor reaches the time limit for a solve/round, the judge stops the solve immediately and records the result as DNF. Exception: Multiple Blindfolded Solving (see [Regulation H1b1](regulations:regulation:H1b1)).
-        - A1a5) A solve is considered to meet the time limit if and only if the final result, after any time penalties are applied, is less than the time limit. Exception: Multiple Blindfolded Solving (see [Regulation H1b1](regulations:regulation:H1b1)).
-    - A1b) If the time limit for a solve is greater than 10 minutes, a stopwatch must be used for timekeeping.
+    - A1a) The organization team applies exactly one kind of time limit (time limit per solve or cumulative time limit) to each round.
+        - A1a1) The default time limit is 10 minutes for each attempt in a round, though the organization team may announce a higher or lower time limit per attempt. In this case, the limit must be the same for all attempts in a round.
+        - A1a2) For events other than 3x3x3 Fewest Moves and 3x3x3 Multi-Blind, the organization team may define a cumulative time limit. The cumulative time limit must either apply to all attempts in one round (e.g. 3 attempts with a cumulative time limit of 20 minutes), or to all attempts in a combination of rounds for different events. In this case, the time limit for a particular attempt is the cumulative time limit minus the total time in other attempts counting towards the time limit so far (see [Regulation A1a5](regulations:regulation:A1a5)). If a round has an applicable cumulative time limit, it must have at most one.
+        - A1a3) The time limits for all rounds announced before the competition, and should not be changed after any affected rounds have begun. Changes must be made at the discretion of the WCA Delegate, who must carefully consider the fairness of the change.
+        - A1a4) The competitor must end each solve before the time limit for the attempt is reached. If a competitor's timer reaches the time limit for an attempt, the judge stops the solve immediately and records the result as DNF. In the case of a cumulative time limit, all remaining attempts with same applicable time limit are recorded as DNS. Exception: Multiple Blindfolded Solving (see [Regulation H1b1](regulations:regulation:H1b1)).
```

Comment:
Should the exception be before the last sentence to be clear that it only applies to that part of the regulation? MBLD cannot have a cumulative time limit to begin with.

Reply:
I'll go one step further: why don't we move "In the case of a cumulative time limit, all remaining attempts with same applicable time limit are recorded as DNS." to a nested Regulation below A1a4)?

Reply:
> I'll go one step further: why don't we move "In the case of a cumulative time limit, all remaining attempts with same applicable time limit are recorded as DNS." to a nested Regulation below A1a4)?

We've never nested beyond 4 levels, and I don't feel this is an important enough case to warrant starting that. :-/

> Should the exception be before the last sentence to be clear that it only applies to that part of the regulation? MBLD cannot have a cumulative time limit to begin with.

Yeah, I think I should just take it out. A1a2) covers it now.